### PR TITLE
sourced ~/.cs50/language

### PR DESCRIPTION
**Pull request by kzidane**

### CODE REVIEW

File: postinst [hunk 1]
```diff
@@ -104,7 +104,11 @@ if ! grep -qs "alias cd" /home/ubuntu/.bashrc; then
     echo 'alias cd="HOME=~/workspace cd"' >>/home/ubuntu/.bashrc
 fi
 
-if ! grep -qs 'prompt' /home/ubuntu/.bashrc; then
+if [ -f /home/ubuntu/.cs50/language ] && ! grep -qs '/home/ubuntu/.cs50/language' /home/ubuntu/.bashrc; then
+    echo "source /home/ubuntu/.cs50/language" >>/home/ubuntu/.bashrc
```
Comment (dmalan) [inline postinst]: Stylistically, I'd put a space after `>>` for clarity.

**Comment by dmalan** [comment]
Where is `~/.cs50/language` created?

File: postinst (hunk 1 above)
Comment (kzidane) [inline postinst]: Was in so many places. Was just trying to be consistent. Fixed per 1b0bcda8.

**Comment by kzidane** [comment]
`~/.cs50/language` is created by cs50/harvard.cs50.simple per https://github.com/cs50/harvard.cs50.simple/pull/85.

**Comment by dmalan** [comment]
Rather than output to .bashrc conditionally, let's just output an if
construct to .bashrc so it only sources if the file exists? More robust.

**Comment by kzidane** [comment]
Generally `.bahsrc` shouldn't be absent, but `>>` creates the file if it doesn't exist?

**Comment by dmalan** [comment]
No no, I mean, in .bashrc:

if language exists
  source language
fi

**Comment by kzidane** [comment]
https://github.com/cs50/ide50/pull/116/files#diff-d87e2a0053f70a97655510229c237b30R107?

**Comment by kzidane** [comment]
@dmalan 4aea1ff.

**Pull request merged**